pkg/ui/page/projects.go

```diff
@@ -112,6 +142,18 @@ func (p *ProjectList) fillUsage() {
 	p.usage.SetText(projectListUsage)
 }
 
+func (p *ProjectList) setFilter(term string) {
+	p.projectListFilter = term
+}
+
+func (p *ProjectList) getFilter() string {
```

Comment:
Not required, it's a private field.

Reply:
You're right! Will rebase and force-push!

Reply:
Actually, I'm not gonna rebase and force push, because then this converstation will make no sense to anyone from the outside.
I'll make this separate commits referencing your comment in this _discussion_.